plugins/report-portal/src/components/LaunchesPage/LaunchesPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {
+  Breadcrumbs,
+  Content,
+  Header,
+  HeaderIconLinkRow,
+  Link,
+  Page,
+  useQueryParamState,
+} from '@backstage/core-components';
+import { useRouteRef } from '@backstage/core-plugin-api';
+
+import AssessmentIcon from '@material-ui/icons/Assessment';
+
+import { projectsRouteRef, rootRouteRef } from '../../routes';
+import { LaunchesPageContent } from './LaunchesPageContent/LaunchesPageContent';
+
+export const LaunchesPage = () => {
+  const rootR = useRouteRef(rootRouteRef);
+  const projectsPage = useRouteRef(projectsRouteRef);
+  const hostName = useQueryParamState('host')[0] as string;
+  const projectName = useQueryParamState('project')[0] as string;
+
+  return (
+    <Page themeId="app">
+      <Header
+        pageTitleOverride="Launches"
+        title={
+          <>
+            <Breadcrumbs>
+              <Link color="inherit" to={rootR()}>
```

Comment:
```suggestion
              <Link color="inherit" to={rootR()}>
```

use complete readable variables

Reply:
changed the var. names here